test/evaluatorHelpers.test.ts

```diff
@@ -252,6 +252,24 @@ describe('renderPrompt', () => {
     expect(renderedPrompt).toBe('Test prompt value1');
     delete process.env.PROMPTFOO_DISABLE_TEMPLATING;
   });
+
+  it('should respect Nunjucks raw tags when variable is provided as a string', async () => {
+    const prompt = toPrompt('{% raw %}{{ var1 }}{% endraw %}');
+    const renderedPrompt = await renderPrompt(prompt, { var1: 'value1' }, {});
+    expect(renderedPrompt).toBe('{{ var1 }}');
+  });
+
+  it('should respect Nunjucks raw tags when no variables are provided', async () => {
+    const prompt = toPrompt('{% raw %}{{ var1 }}{% endraw %}');
+    const renderedPrompt = await renderPrompt(prompt, {}, {});
+    expect(renderedPrompt).toBe('{{ var1 }}');
+  });
+
+  it('should render variables that are template strings', async () => {
+    const prompt = toPrompt('{{ var1 }}');
+    const renderedPrompt = await renderPrompt(prompt, { var1: '{{ var2 }}', var2: 'value2' }, {});
+    expect(renderedPrompt).toBe('value2');
+  });
```

Comment:
oh I see, it's captured here. I wonder if we should even allow 2nd order templates like this? Someone might complain that `var2: {{ var3 }}` doesn't work 😢

Reply:
Let's cross that bridge when we come to it.